static/app/views/insights/browser/resources/views/resourceSummaryPage.tsx

```diff
@@ -47,6 +48,27 @@ function ResourceSummary() {
   const {
     query: {transaction},
   } = useLocation();
+
+  const {openSamplesDrawer} = useSamplesDrawer({
+    Component: (
+      <SampleList
+        transactionRoute={webVitalsModuleURL}
+        subregions={filters[SpanMetricsField.USER_GEO_SUBREGION]}
+        groupId={groupId}
+        moduleName={ModuleName.RESOURCE}
+        transactionName={transaction as string}
```

Comment:
nit: Maybe we could use `decodeScalar` so we don't need this type assertion? might have to handle undefined though or add a fallback

Reply:
Probably yes! I didn't bother changing this since it's not part of the panel change (I'm just moving the code around) but I agree that using `decodeScalar` (probably in `useLocationQuery`) would be better here